paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetCompose.kt

```diff
@@ -94,6 +94,22 @@ fun rememberPaymentSheet(
     }
 }
 
+/**
+ * Creates a [PaymentSheet] that is remembered across compositions and initializes optional callbacks if set
+ *
+ * @param builder which contains required [PaymentSheetResultCallback] as well as other optional callbacks
+ */
+@Composable
+internal fun rememberPaymentSheet(builder: PaymentSheet.Builder): PaymentSheet {
+    builder.createIntentCallback?.let {
```

Comment:
There's quite a few different ways to set these. We set them inside the builder when not using the compose variant. 

Should we cleanup some of this duplicated code to make a single code path to set them?

I think in an ideal world, we'd update the other public composables to use the builders code path internally.

Reply:
For the compose path, they are set using `LaunchedEffect`. Is this required? Also based on @tillh-stripe's comment, I'm considering removing the composable `build` function and make `rememberPaymentSheet(builder: PaymentSheet.Builder)` and `rememberPaymentSheetFlowController(builder: PaymentSheet.FlowController.Builder)` public

Reply:
I'm on board with that.